src/utils/Validator.js

```diff
@@ -0,0 +1,36 @@
+import { ERROR_MESSAGES, createError } from '../constants/constants.js';
+
+class Validator {
+  static numberOfGamesValidation(input) {
+    if (Number.isNaN(input)) {
+      throw createError(ERROR_MESSAGES.NOT_A_NUMBER);
+    }
+    if (input < 0) {
+      throw createError(ERROR_MESSAGES.NEGATIVE_NUMBER);
+    }
+    if (input === 0) {
+      throw createError(ERROR_MESSAGES.ZERO_NUMBER);
+    }
+
+    if (!Number.isInteger(input)) {
+      throw createError(ERROR_MESSAGES.NOT_INTEGER);
+    }
+  }
+
+  static carValidation(carArray) {
+    if (carArray.length === 1) {
```

Comment:
다른 조건들은 변수로 빼두셨는데 이것만 안빼두신 이유가 있으신가요?
저는 변수로 빼두는 이유가 조건이 길어서도 있지만, 변수명을 통해 가독성을 높일 수도 있다고 생각하거든요!
```suggestion
    const hasNoCars = carArray.length === 1
    if (hasNoCars) {
```

Reply:
누락된것같습니다... 알려주셔서 감사합니다!! 꼭 참고하도록 하겠습니다!